templates/bits/igf.tpl

```diff
@@ -6,7 +6,7 @@
 {$placeholder=$placeholder|default:""}
 {$required=$required|default:true}
 {$autofocus=$autofocus|default:false}
-<div class="input-group {if $errors.$field}has-error{/if}">
+<div class="input-group">
```

Comment:
Is this a good idea? The [has-error class](http://getbootstrap.com/css/#forms-control-validation) highlights fields with errors. I'm sure some calls to `igf.tpl` use this class.